build-docs-rsync

```diff
@@ -6,17 +6,34 @@ GIT_RESET="git reset --quiet --hard origin/master"
 cd /local/repos/phpdoc/phd
 $GIT_FETCH && $GIT_RESET >/tmp/log-phd 2>&1
 
-# Update docs sources
-cd /local/repos/phpdoc
-svn up >/tmp/log-svn 2>&1
+BASE_DIR=/local/repos/phpdoc-git
+
+assertUpToDateRepo() {
+    if [ ! -d $BASE_DIR/$1 ]; then
+        git clone https://git.php.net/repository/doc/$2.git $BASE_DIR/$1
+    fi
+    git -C $BASE_DIR/$1 fetch origin && git -C $BASE_DIR/$1 reset --hard origin/master > /tmp/log-git 2>&1
```

Comment:
```suggestion
    git -C $BASE_DIR/$1 fetch origin && git -C $BASE_DIR/$1 reset --hard origin/master >> /tmp/log-git 2>&1
```
To avoid emptying the log for each assertUpToDateRepo call? Though then we probably should also explicitly clear the log at the start of the script...

Reply:
Originally I was thinking of using `logger` to append the messages to the system log and also make use of log turnover etc. but refrained from it to make it as close to the previous version as possible but didn't take into account that it will clear the log every time it runs...

Reply:
For what it's worth, clearing the log on each run was the old behaviour so you're not doing anything worse here.